packages/katello/rubygem-foreman_rh_cloud/rubygem-foreman_rh_cloud.spec

```diff
@@ -115,6 +115,8 @@ cp -a .%{gem_dir}/* \
 %exclude %{gem_cache}
 %{gem_spec}
 %{foreman_bundlerd_plugin}
+%{foreman_apipie_cache_foreman}
+%{foreman_apipie_cache_plugin}
```

Comment:
These two need to be re-dropped. They were removed with the drop of apipie:cache support.